Dockerfile.dapper

```diff
@@ -14,17 +16,37 @@ ENV KUSTOMIZE_VERSION v4.4.1
 # kontainer-driver-metadata branch to be set for specific branch other than dev/master, logic at rancher/rancher/pkg/settings/setting.go
 ENV CATTLE_KDM_BRANCH=dev-v2.6
 
-RUN zypper -n install gcc binutils glibc-devel-static ca-certificates git-core wget curl unzip tar vim less file xz gzip sed gawk iproute2 iptables jq
+RUN zypper -n install -y --replacefiles \
+    gcc \
+    binutils \
+    glibc-devel-static \
+    ca-certificates \
+    git-core \
+    wget \
+    curl \
+    unzip \
+    tar \
+    vim \
+    less \
+    file \
+    xz \
+    gzip \
+    sed \
+    gawk \
+    iproute2 \
+    iptables \
+    jq
 # use containerd from k3s image, not from bci
 RUN zypper install -y -f docker && rpm -e --nodeps --noscripts containerd
 
-RUN curl -sLf https://github.com/rancher/machine/releases/download/${CATTLE_MACHINE_VERSION}/rancher-machine-${ARCH}.tar.gz | tar xvzf - -C /usr/bin
+RUN curl -sLf https://github.com/rancher/machine/releases/download/${CATTLE_MACHINE_VERSION}/rancher-machine-${DAPPER_HOST_ARCH}.tar.gz | tar xvzf - -C /usr/bin
```

Comment:
Aren't these the same?

Reply:
Same. Not sure why I changed that. Now reverted.